src/Components/Notes/NoteCardCreate.jsx

```diff
@@ -0,0 +1,187 @@
+   *
+   */
+  function isCreateNoteActive() {
+    // TODO(Oleg) noteIndex is used for note number :: need to introduce internal index system
+    let noteIndex
+    if (createdNotes) {
+      noteIndex = Object.keys(createdNotes).length
+    } else {
+      noteIndex = 1
+    }
+    const issuePayload = {
+      title,
+      body,
+    }
+    postIssue(repository, issuePayload, accessToken)
+    toggleIsCreateNoteActive()
+    const localNote = {
+      index: noteIndex,
+      id: noteIndex,
+      number: noteIndex,
+      title: title,
+      body: body,
+      date: new Date().toISOString(),
+      username: user ? user.name : 'username',
+      avatarUrl: user ? user.picture : '',
+      numberOfComments: '',
+      synchedNote: false,
+    }
+    if (createdNotes === null) {
+      setCreatedNotes([localNote])
+    } else {
+      setCreatedNotes([localNote, ...createdNotes])
+    }
+  }
+
+  return (
+    <Paper
+      elevation={1}
+      variant='note'
+      square
+      sx={{
+        width: '100%',
+
+      }}
+    >
+      <CardHeader
+        title={
+          <InputField
+            placeholder={'Note Title'}
+            inputText={title}
+            setInputText={setTitle}
+            multiline={false}
+            maxLength={256}
+          />}
+        avatar={
+          isAuthenticated ?
+          <Avatar
+            alt={user.name}
+            src={user.picture}
+            sx={{width: 22, height: 22}}
+          /> :
+          <Avatar alt={username} src={avatarUrl}/>
+        } sx={{
+          backgroundColor: theme.palette.primary.main,
+        }}
+      />
+      <CardContent
+        sx={{
+          'padding': '0px 20px 0px 20px',
+          'margin': '0px 0px 0px 0px',
+          'minHeight': '100px',
+          '& img': {
+            width: '100%',
+          },
+        }}
+      >
+        <Box
+          sx={{
+            margin: '10px 0px',
+          }}
+        >
+          <InputField
+            placeholder={'Note Body'}
+            inputText={body}
+            setInputText={setBody}
+            multiline={true}
+            maxLength={65000}
+          />
+        </Box>
+      </CardContent>
+      <CardActions isCreateNoteActive={isCreateNoteActive}/>
+    </Paper>
+  )
+}
+
+
+const CardActions = ({isCreateNoteActive}) => {
+  const theme = useTheme()
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'flex-end',
+        alignItems: 'center',
+        padding: '0px 14px 0px 14px',
+        height: '50px',
+        backgroundColor: theme.palette.primary.main,
+      }}
+    >
+      <TooltipIconButton
+        title='Submit'
+        size='small'
+        placement='bottom'
+        onClick={isCreateNoteActive}
+        icon={<Submit style={{width: '15px', height: '15px'}}/>}
+      />
+    </Box>
+  )
+}
+
+
+/**
+ * Input
+ *
+ * @param {string} placeholder input placeholder
```

Comment:
when the param is in an object, I believe `@property` is correct for tsc.

Also looks like last couple of params are missing jsdocs